imports/plugins/core/graphql/server/resolvers/account/query/getUserId.js

```diff
@@ -1,5 +1,7 @@
+import { Meteor } from "meteor/meteor";
+
 export default function getUserId(_, __, context) {
   const userId = context && context.user && context.user._id;
-  if (!userId) throw new Error("access-denied");
+  if (!userId) throw new Meteor.Error("access-denied");
```

Comment:
Do we need the longer version of the Error here too? Or is the short version sufficient for something like this?

Reply:
That's a good point, but I'm not going to fix it because I'm going to suggest we remove this query anyway. It duplicates the viewer query. Will get this merged and then remove it in a future PR soon.